package.json

```diff
@@ -135,7 +136,10 @@
   "dependencies": {
     "@twilio/webrtc": "4.3.2",
     "backoff": "^2.5.0",
+    "chromedriver": "^87.0.5",
```

Comment:
@PikaJoyce - These dependencies should not go here.  We do have a step to install these as part of `test:framework:instsall' - We removed these packages from `devDependancies` to avoid downloading these as part of every circleci job.

Reply:
Weird this got left in, had a conflict but this is now fixed.